core/src/main/java/fr/sncf/osrd/api/WorkerLoadEndpoint.java

```diff
@@ -37,18 +40,22 @@ public Response act(Request req) {
         }
     }
 
-    public static final class InfraLoadRequest {
+    public static final class WorkerLoadRequest {
         /** Infra id */
         public String infra;
 
         /** Infra version */
         @Json(name = "expected_version")
         public int expectedVersion;
 
+        /** Timetable ID */
+        public Integer timetable;
+
         /** Create InfraLoadRequest */
-        public InfraLoadRequest(String infra, int expectedVersion) {
+        public WorkerLoadRequest(String infra, int expectedVersion, Integer timetable) {
```

Comment:
Honestly, if you know how, we could convert this file to kotlin, which would allow us to add a default null value for the timetable (which would mean you wouldn't need to change some of the uses of this class):
```suggestion
        data class WorkerLoadRequest(
           infra: String,
           expectedVersion: int,
           timetable: Int? = null
        )
```

Reply:
I did not check how to convert this file in kotlin, so I will leave it this way 😕